docs/integrations/sources/metabase.md

```diff
@@ -77,6 +97,7 @@ The Metabase source connector supports the following [sync modes](https://docs.a
 
 | Version | Date       | Pull Request                                             | Subject                                                                                                                          |
 | :------ | :--------- | :------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------- |
+| 2.2.0   | 2024-08-27 | 
```

Comment:
This is a breaking change as old connections will need to update their config in the UI please bump accordly.

Reply:
This makes sense. I actually want an opinion on another thing.

Currently, Metabase source uses the `LegacySessionTokenAuthenticator`, my PR updates it to use the new `SessionTokenAuthenticator`. Do we prefer to keep the old one or should we use the new one?

Also, I realized I didn't bring it up here in the PR but in [Slack](https://airbytehq.slack.com/archives/C054V9JFTC6/p1725177930769979), I was having trouble with `SelectiveAuthenticator` to pass down some properties to the `SessionTokenAuthenticator`.